jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -260,5 +286,5 @@ function get_host_addr() {
   hostname -i | sed -e '1!d; s/[[:space:]].*//'
 }
 
-
+trap finish EXIT
```

Comment:
Install the exit handler as early as possible, ideally define and install the exit handler at the very beginning of the script.

Reply:
done